src/Benchmarks/Program.cs

```diff
@@ -133,6 +140,21 @@ private static int GetThreadCount(IConfigurationRoot config)
             var threadCountValue = config["threadCount"];
             return threadCountValue == null ? -1 : int.Parse(threadCountValue);
         }
+
+        private static IConnectionFilter GetConnectionFilter(IConfigurationRoot config, IConnectionFilter prevFilter)
+        {
+            var connectionFilterValue = config["connectionFilter"];
+            if (string.IsNullOrEmpty(connectionFilterValue))
+            {
+                return null;
+            }
+            else
+            {
+                var connectionFilterType = Type.GetType(connectionFilterValue, throwOnError: true);
```

Comment:
Do you envision us having different connection filters with no arguments? Why not make it a boolean flag? On or off?